.github/workflows/main.yml

```diff
@@ -0,0 +1,138 @@
+name: main
+'on':
+  - workflow_dispatch
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Generate Dashboard
+        id: dashboard
+        uses: ethomson/issue-dashboard@v1
+        with:
+          config: |
+            title: 'Health Dashboard'
+            output:
+              format: html
+              filename: 'main.html'
+            setup: | 
+                  const repos = 'repo:Azure/login repo:Azure/webapps-deploy repo:Azure/aci-deploy repo:build-vm-image repo:Azure/deploy-to-azure-cli-extension repo:Azure/functions-action repo:Azure/functions-container-action repo:Azure/mysql repo:Azure/postgresql repo:Azure/powershell repo:Azure/sql-action'
+                  userdata.count = async function(itemType,label) {
+                    const state = (label==="closed")? "closed" : "open"
```

Comment:
nit: formatting doesn't look proper (spacing) Can you please format the whole file